exporters/css/src/files/style-file.ts

```diff
@@ -1,32 +1,55 @@
-import { FileHelper } from "@supernovaio/export-helpers"
-import { OutputTextFile, Token, TokenGroup, TokenType } from "@supernovaio/sdk-exporters"
-import { exportConfiguration } from ".."
-import { convertedToken } from "../content/token"
+import { FileHelper } from '@supernovaio/export-helpers';
+import {
+  OutputTextFile,
+  Token,
+  TokenGroup,
+  TokenType,
+} from '@supernovaio/sdk-exporters';
+import { exportConfiguration } from '..';
+import { convertedToken } from '../content/token';
 
-export function styleOutputFile(type: TokenType, tokens: Array<Token>, tokenGroups: Array<TokenGroup>): OutputTextFile | null {
+export function styleOutputFile(
+  type: TokenType,
+  tokens: Array<Token>,
+  tokenGroups: Array<TokenGroup>
+): OutputTextFile | null {
   // Filter tokens by top level type
-  const tokensOfType = tokens.filter((token) => token.tokenType === type)
+  const tokensOfType = tokens.filter((token) => token.tokenType === type);
+
+  // Only export supported token types
+  // TODO: Add all token types
+  const supportedTypes: Array<TokenType> = [TokenType.color];
+  if (!supportedTypes.includes(type)) {
+    return null;
+  }
 
   // Filter out files where there are no tokens, if enabled
   if (!exportConfiguration.generateEmptyFiles && tokensOfType.length === 0) {
-    return null
+    return null;
   }
 
   // Convert all tokens to CSS variables
-  const mappedTokens = new Map(tokens.map((token) => [token.id, token]))
-  const cssVariables = tokensOfType.map((token) => convertedToken(token, mappedTokens, tokenGroups)).join("\n")
+  const mappedTokens = new Map(tokens.map((token) => [token.id, token]));
+  const cssVariables = tokensOfType
+    .map((token) => convertedToken(token, mappedTokens, tokenGroups))
+    .join('\n');
+
+  // Custom stylelint rules
+  const contentRules = '/* stylelint-disable color-hex-length */\n';
```

Comment:
Fant ingen enkel config-endring for å sette dynamisk lengde på hex-verdiene for farger (#fff istedenfor #ffffff f.eks.), så disabler heller regelen for tokens-filer